src/main/java/pl/wip/wipplatformbe/models/User.java

```diff
@@ -0,0 +1,52 @@
+package pl.wip.wipplatformbe.models;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import pl.wip.wipplatformbe.enums.Badge;
+import pl.wip.wipplatformbe.enums.Role;
+import pl.wip.wipplatformbe.enums.UserStatus;
+//import org.springframework.security:spring-security-crypto
+//import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Document
+public class User {
+
+    @Id
+    private String id;
+    private String psswrdHashed;
+    private String email;
+    private Photo profilePicture;
+    private String username;
+    private String firstName;
+    private String lastName;
+    private List<Role> role;
+    private List<Badge> badges;
+    private String description;
+    private Date createdAt;
+    private UserStatus status;
+
+
+
+
+
+    public User() {}
+
+    // not sure about profilePicture
```

Comment:
Why? Is there something missing from the diagram?

Reply:
Work comments, shouldn't have been passed to repo